src/components/Home/Home.js

```diff
@@ -1,30 +1,35 @@
-import React, { useEffect, useState } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import { Context as SearchContext } from '../../contexts/SearchContext';
 import { getYoutubeMainInfoList } from '../../services/youtube';
 import { CardInfo } from './CardInfo/CardInfo';
 import { CardContainer, Title } from './styles';
 
-export const Home = () => {
+export const Home = ({ handleViewMore }) => {
+  const {
+    state: { searchText },
+  } = useContext(SearchContext);
   const [youtubeInfoList, setYoutubeInfoList] = useState([]);
 
   useEffect(() => {
     const getList = async () => {
-      const list = await getYoutubeMainInfoList();
+      const list = await getYoutubeMainInfoList(searchText);
       setYoutubeInfoList(list);
     };
 
     getList();
-  }, []);
+  }, [searchText]);
 
   return (
     <>
-      <Title>Welcome to my first challenge!!!</Title>
+      <Title>Welcome to my challenge 3!!!</Title>
       <CardContainer>
         {youtubeInfoList.map((item) => (
```

Comment:
you can destructure properties:
`youtubeInfoList.map(({ id, title, description, imageURL }) => (...`